commit ea0af4b6d681b4b6b8c65bb2ce4c04f5aaed2e54
Author: Kamal Heib <kamalheib1@gmail.com>
Date:   Wed Jan 17 16:52:24 2018 +0200

    staging: greybus: authentication.c: Fix alignment should match open parenthesis
    
    Fix the following error found by checkpatch.pl:
    CHECK: Alignment should match open parenthesis
    +static int cap_ioctl(struct gb_cap *cap, unsigned int cmd,
    +                        void __user *buf)
    
    Signed-off-by: Kamal Heib <kamalheib1@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/staging/greybus/authentication.c b/drivers/staging/greybus/authentication.c
index 16cc65e1472b..a5d7c53df987 100644
--- a/drivers/staging/greybus/authentication.c
+++ b/drivers/staging/greybus/authentication.c
@@ -202,7 +202,7 @@ static int cap_release(struct inode *inode, struct file *file)
 }
 
 static int cap_ioctl(struct gb_cap *cap, unsigned int cmd,
-			 void __user *buf)
+		     void __user *buf)
 {
 	struct cap_ioc_get_endpoint_uid endpoint_uid;
 	struct cap_ioc_get_ims_certificate *ims_cert;